README.md

```diff
@@ -12,6 +12,8 @@ Whether you are new to the process and are building your first AI assistant or y
 - How do I improve my assistant’s ability to understand questions?
 
 ## Usage
+If you clone the notebook from this repository, please follow the following steps. For usage in Watson studio, please refer to the 
```

Comment:
Rephrasing a bit: `clone the notebook from this repository locally, please use the steps bellow.`